test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -55,10 +54,11 @@ function mixing_length(
     Shear² = diffusive.turbconv.S²
     tke_en = max(en.ρatke, 0) * ρinv / env.a
 
+    # buoyancy related functions
+    # compute obukhov_length and ustar from SurfaceFlux.jl here
     ustar = m.turbconv.surface.ustar
     obukhov_length = m.turbconv.surface.obukhov_length
 
-    # buoyancy related functions
     ∂b∂z, Nˢ_eff = compute_buoyancy_gradients(m, state, diffusive, aux, t, ts)
     Grad_Ri = ∇Richardson_number(∂b∂z, Shear², 1 / ml.max_length, ml.Ri_c)
     Pr_z = turbulent_Prandtl_number(ml.Pr_n, Grad_Ri, ml.ω_pr)
```

Comment:
Since variables are defined pointwise in the climate machine, maybe we can omit z here and call it `Pr_t`, for turbulent.